cmd/weaver/output.go

```diff
@@ -12,6 +12,7 @@ import (
 type output struct {
 	FunctionName string
 	Args         []outputArg
+	ProcInfo     procInfo `json:"procInfo,omitempty"`
```

Comment:
I'm wary of putting `omitempty` here. If the value is empty then reading it from the perf buffer/unmarshalling it will fail. Seems like it could be a place to hide unwanted behavior.

Also I think i'm fine with having uppercase "ProcInfo", is it more idiomatic to start with lowercase? I'm totally open to your advice and opinion here. If you think lowercase is better, can you add json struct tags to all the fields?

Reply:
Hmm I see, will delete this `omitempty`.

I don't think there is a convention for json key naming, but if we follow camelCase format it should start with lowercase I think. I can add json tags to all fields if you're ok with that.

Reply:
Yea I suppose so, json tags would be great in that case if you don't mind :)

Reply:
done